schemas/test/en/test_theme_health.json

```diff
@@ -21,6 +21,9 @@
             }
         }
     },
+    "post_submission": {
+        "view_response": true
+    },
```

Comment:
Lets add this to social for consistency.

Reply:
Added, thanks.